test/r/0.migrations.spec.ts

```diff
@@ -36,89 +36,89 @@ import torus_json             from "./jsons/r3fJsons/elements/torus.json";
 import videoFlat_json         from "./jsons/r3fJsons/elements/videoFlat.json";
 import { VarCategory } from "../../src/r/definitions/variables";
 
-describe("r Migrations", () => {
-  xit("should test r migration", () => {
-    const r100 = migrateProjectRJson(safehands_r0, 100);
-    // const r101 = migrateProjectRJson(safehands_r0, 101);
-    // const r107 = migrateProjectRJson(accenture_r0);
-    // const emptyProject = migrateProjectRJson({});
-    // // fs.writeFileSync("./test/r/jsons/accenture.json",JSON.stringify(r107));
+// describe("r Migrations", () => {
```

Comment:
revert changes here. If you need to test a single file, then change the `.mocharc.jsonc` file to include only your test file